### DISCUSSION TITLE
SVG support missing

### DISCUSSION THREAD

**Discussion by fate8383** [nextcloud/all-in-one#2179] [Questions] [answered]
There are some warnings regarding your setup.
Module php-imagick in this instance has no SVG support. For better compatibility it is recommended to install it.

I have above warning in administration overview, how do I correct this?

**Comment by Zoey2936** [comment]
do you run nc v26?

**Reply by fate8383** [reply]
> do you run nc v26?

no
[nextcloud] 25.0.4
Nextcloud AIO v4.6.2

**Reply by Zoey2936** [reply]
did you tried to restart aio?

**Reply by dantefromhell** [reply]
I see the same error on nextcloud-aio v6.2.1 / nextcloud 26.0.3

Restarting containers did not help

**Comment by fate8383** [comment]
just aio or all containers?

**Reply by Zoey2936** [reply]
all

**Reply by fate8383** [reply]
that solved it, thanks

**Reply by frake65** [reply]
Has there been a regression regarding this issue introduced with the actual v7.12.0 beta release of AIO? - The mentioed warning message comes up for me.

**Reply by szaimen** [reply]
Yes, this will be fixed with the next beta release.

**Reply by szaimen** [reply]
This is now released with v7.12.1 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by frake65** [reply]
works for me now, thank you!

**Comment by szaimen** [comment, marked as answer]
https://github.com/nextcloud/all-in-one/discussions/2179#discussioncomment-5392288
